README.md

```diff
@@ -2,7 +2,7 @@
 
 > A React + Redux framework for creating CRUDs
 
-[![NPM](https://img.shields.io/npm/v/croods.svg)](https://www.npmjs.com/package/croods) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![NPM](https://img.shields.io/npm/v/croods.svg)](https://www.npmjs.com/package/croods) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com) [![CircleCI](https://circleci.com/gh/danielweinmann/croods.svg?style=svg)](https://circleci.com/gh/danielweinmann/croods) [![Maintainability](https://api.codeclimate.com/v1/badges/1fbb6da18b9ce2664baa/maintainability)](https://codeclimate.com/github/danielweinmann/croods/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/1fbb6da18b9ce2664baa/test_coverage)](https://codeclimate.com/github/danielweinmann/croods/test_coverage)
```

Comment:
You can remove the style guide badge: we're not following this style :P this README was generated by `create-react-library`.